### ISSUE TITLE
[Core] - LFG improper scaling

### ISSUE THREAD

**Issue by taktrix** [GuabinaCore/WoWKoi#1289]
Problem: When queing up in random LFG you get dungeons that are way below level of what you actually are and the scaling is improper. I qued up at level 45 and got a que for deadmines, the tank was level 19, two of us were in our mid 40s and the two other people were around level 28. So I thought "ok maybe the levels stayed the same but the stats all evened out. Nope, the tank only ended up with around 2000HP while we were at around 7000hp. 

Solution: Adjust scaling to an average between all levels and set stats to set average.

**Comment by Serithul** [comment]
There is no bug present. For every person in the team the mob level is different due to level scaling. All dmg and healing should be scaling up or down depending on the source and the destination target.

What you are suggested as a solution is called party sync and Blizzard added it on 8.2.5 .

**Comment by taktrix** [comment]
Sounds like they found the solution, because clearly the scaling was miles off, the level 19 tank was getting 3 shot by mobs that weren't scaling down to his level. And there are actually a lot of instances of scaling being miles off in this game. I just ran into arathi highlands where I found myself fighting a mob that was level skull. And if I run through starter areas some mobs will be my level but other mobs will be like level 20.
